elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStore.java

```diff
@@ -1,33 +1,21 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
 package com.yahoo.elide.datastores.aggregation;
 
 
 import com.yahoo.elide.core.DataStore;
 import com.yahoo.elide.core.DataStoreTransaction;
 import com.yahoo.elide.core.EntityDictionary;
 import com.yahoo.elide.core.RequestScope;
-import com.yahoo.elide.core.filter.expression.FilterExpression;
-import com.yahoo.elide.core.pagination.Pagination;
-import com.yahoo.elide.core.sort.Sorting;
-import com.yahoo.elide.datastores.aggregation.annotation.MetricAggregation;
-import com.yahoo.elide.datastores.aggregation.dimension.Dimension;
-import com.yahoo.elide.datastores.aggregation.dimension.TimeDimension;
-import com.yahoo.elide.datastores.aggregation.engine.schema.SQLSchema;
-import com.yahoo.elide.datastores.aggregation.filter.visitor.SplitFilterExpressionVisitor;
-import com.yahoo.elide.datastores.aggregation.metric.AggregatedMetric;
-import com.yahoo.elide.datastores.aggregation.metric.Aggregation;
-import com.yahoo.elide.datastores.aggregation.metric.Metric;
-import com.yahoo.elide.datastores.aggregation.metric.Sum;
 import com.yahoo.elide.datastores.aggregation.schema.Schema;
-import com.yahoo.elide.request.Attribute;
 import com.yahoo.elide.request.EntityProjection;
-import sun.awt.image.ImageWatched;
-
-import java.sql.Time;
-import java.util.LinkedHashSet;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
 
+/**
+ * Aggregation Data Store translates given information into query which can be used by a {@link QueryEngine} to return back results.
+ */
```

Comment:
Does AggregationDataStore just **translates** queries? I think something like "DataStore that supports aggregation functionality" would be better for this class.